articles/quickstart.md

```diff
@@ -96,26 +108,24 @@ Our goal is to prepare two qubits in a [Bell state](https://en.wikipedia.org/wik
     }
     ```
 
-    This operation may now be called to set a qubit to a known state (`Zero` or `One`).
+    This operation may now be called to set a qubit to a classical state, either returning `Zero` 100% of the time or returning `One` 100% of the time.  `Zero` and `One` are constants that represent the only two possible results of a measurement of a qubit.
 
-    We measure the qubit, if it's in the state we want, we leave it alone, otherwise, we flip it with the `X` gate.
+    The operation `Set` measures the qubit, if it's in the state we want, we leave it alone, otherwise, by executing the `X` operation, we change the qubit state to the desired state.
 
 ### About Q# operations
 
-An operation is the basic unit of quantum execution in Q#. It is roughly equivalent to a function in C or C++ or Python, or a static method in C# or Java.
+A Q# operation is a quantum subroutine. That is, it is a callable routine that contains quantum operations.
 
 The arguments to an operation are specified as a tuple, within parentheses.
 
 The return type of the operation is specified after a colon. In this case, the `Set` operation has no return, so it is marked as returning `Unit`. This is the Q# equivalent of `unit` in F#, which is roughly analogous to `void` in C#, and an empty tuple (`Tuple[()]`) in Python.
 
-### About quantum gates
-
-You have used two quantum gates in your first Q# operation:
+You have used two quantum operations in your first Q# operation:
 
-* The [M](xref:microsoft.quantum.intrinsic.m) gate, which measures the state of the qubit
-* The [X](xref:microsoft.quantum.intrinsic.x) gate, which flips the state of a qubit
+* The [M](xref:microsoft.quantum.intrinsic.m) operation, which measures the state of the qubit
+* The [X](xref:microsoft.quantum.intrinsic.x) operation, which flips the state of a qubit
 
-A quantum gate transforms the state of a qubit. The name comes from the analogy of a classical logic gate such as a `NOT` gate, which inverts the state of a classical bit.
+A quantum operation transforms the state of a qubit. Sometime people talk about quantum gates instead of operations, in analogy to classical logic gates. This is rooted in the early days of quantum computing when algorithms were merely a theoretical construct and visualized as diagrams similarly to circuit diagrams in classical computing.
```

Comment:
👍

Reply:
Thumbs up to Bettina for the suggested prose!